### DISCUSSION TITLE
Dependabot and Forks?

### DISCUSSION THREAD

**Discussion by srl295** [community/community#14893] [Code Security]
Hello,
 https://github.com/dependabot/dependabot-core/issues/2804 and linked issues have been around since 2019. Is there a timeline for some way to disable dependabot in forks? Thanks.

**Comment by hotate29** [comment]
I feel the same way. I hope to see some progress on this issue published in the near future!

EDIT: Oops, maybe it wasn't very appropriate to write my personal opinion here...
